### PR TITLE
Make Experimentors spawn base turn a game rule

### PR DESCRIPTION
**Pull request by o01eg**

### CODE REVIEW

File: default/scripting/game_rules.focs.py [hunk 1]
```diff
@@ -56,6 +56,16 @@
...
 )
 
+GameRule(
+    name="RULE_EXPERIMENTORS_SPAWN_BASE_TURN",
+    description="RULE_EXPERIMENTORS_SPAWN_BASE_TURN_DESC",
+    category="CONTENT",
+    type=int,
+    default=250,
+    min=0,
+    max=300,
```
Comment (geoffthemedio) [inline default/scripting/game_rules.focs.py]:
```suggestion
    max=500,
```

File: default/stringtables/en.txt [hunk 2]
```diff
@@ -1213,6 +1213,12 @@ Enable Experimentors
 RULE_ENABLE_EXPERIMENTORS_DESC
 Enables Experimentors during galaxy generation
 
+RULE_EXPERIMENTORS_SPAWN_BASE_TURN
+Experimentors spawn base turn
+
+RULE_EXPERIMENTORS_SPAWN_BASE_TURN_DESC
+Base turn when Experimentors starts to spawn
```
Comment (geoffthemedio) [inline default/stringtables/en.txt]: should probably explain that a bit more... not clear what the "base turn" means.

Comment (o01eg) [reply default/stringtables/en.txt]: I'm not sure either, it base of quite complex formula to calculate actual turn to start spawning.

File: default/scripting/game_rules.focs.py (hunk 1 above)
Comment (o01eg) [reply default/scripting/game_rules.focs.py]: Fixed.

File: default/stringtables/en.txt (hunk 2 above)
Comment (MichaelCourtney) [reply default/stringtables/en.txt]: Minimum turn experimentors spawn. Modified by galaxy size and AI Agression?

Comment (o01eg) [reply default/stringtables/en.txt]: I've used that phase.

**Comment by o01eg** [comment]
Implemented in #4552

**Review by MichaelCourtney — COMMENTED** [review]
Looks ok to me. Up to Geoff

**Pull request merged**